packages/lit-auth-client/src/lib/providers/BaseProvider.ts

```diff
@@ -161,37 +165,31 @@ export abstract class BaseProvider {
 
         let response: SignSessionKeyResponse;
 
+        // common data for the signSessionKey function call
+        const commonData = {
+          sessionKey: sessionKey,
+          statement: authCallbackParams.statement,
+          pkpPublicKey: params.pkpPublicKey,
+          expiration: authCallbackParams.expiration,
+          resources: authCallbackParams.resources,
+          chainId: chainId,
+          ...(params.resourceAbilityRequests && {
+            resourceAbilityRequests: params.resourceAbilityRequests,
+          }),
+        };
+
         if (params.authMethod.authMethodType === AuthMethodType.EthWallet) {
           const authSig = JSON.parse(params.authMethod.accessToken);
+
           response = await nodeClient.signSessionKey({
-            statement: authCallbackParams.statement,
-            sessionKey: params.sessionSigsParams.sessionKey,
-            authMethods: [],
+            ...commonData,
             authSig: authSig,
```

Comment:
We no longer pass AuthSig. I guess it's removed in another PR?

Reply:
this will be removed in #444